app/search_builders/curation_concerns/search_builder.rb

```diff
@@ -31,22 +31,4 @@ def discovery_permissions
     return ['edit'] if blacklight_params[:works] == 'mine'
     super
   end
-
-  # This is included as part of blacklight search solr params logic
-  def filter_models(solr_parameters)
-    solr_parameters[:fq] ||= []
-    solr_parameters[:fq] << '(' + (work_clauses + collection_clauses).join(' OR ') + ')'
-  end
-
-  def work_clauses
-    return [] if blacklight_params.key?(:f) && Array(blacklight_params[:f][:generic_type_sim]).include?('Collection')
```

Comment:
Yeah, i'm almost certain we want to drop `only_curation_concerns` which was only introduced in https://github.com/projecthydra-labs/worthwhile/commit/5cb4ccd85a5d0732c3cbdeda14fd7a5614011e10 (as `only_generic_files_and_curation_concerns`)